soh/soh/Extractor/Extract.cpp

```diff
@@ -479,7 +479,8 @@ bool Extractor::Run(std::string searchPath, RomSearchMode searchMode) {
                     ShowCrcErrorBox();
                 } else {
                     ShowErrorBox("Rom CRC invalid",
-                                 "Rom CRC did not match the list of known good roms. Trying the next one...");
+                                 "Rom CRC did not match the list of known good roms. Trying the next one...\n\n"
+                                 "Check if your ROM is valid at https://ship.equipment/");
```

Comment:
Maybe "compatible" instead of "valid"? It might also be nice to mention "and see a list of compatible versions" or something like that, but I couldn't find a way to word it that I liked.